tracker/classify.go

```diff
@@ -0,0 +1,42 @@
+// Package tracker implements an object tracker as a Viam vision service
+// This file contains methods that are useful for classifying the detections
+package tracker
+
+import (
+	"context"
+	"image"
+
+	"go.viam.com/rdk/rimage"
+	"go.viam.com/rdk/services/vision"
+	objdet "go.viam.com/rdk/vision/objectdetection"
+)
+
+// method will take a slice of tracks and return a slice of tracks.
+// The only difference is that the track will now include the detection classification
+func classifyTracks(ctx context.Context, tracks []*track, img image.Image, classifier vision.Service) []*track {
+	
+	for _, tr := range tracks {
+		cropped := cropImageFromDet(img, tr.Det)
+		out, err := classifier.Classifications(ctx, cropped, 1, nil)
+		if err != nil || len(out) > 1 {
```

Comment:
if out is more than 1, couldn't you just grab the top classification? Granted that shouldn't happen. Did you mean to check if len(out) was less than 1? That is the edge case to look for.

And could you also log the error at least from the classifier, so include a logger, within the arguments.